base/complex.jl

```diff
@@ -897,6 +897,10 @@ function complex(A::AbstractArray{T}) where T
     convert(AbstractArray{typeof(complex(zero(T)))}, A)
 end
 
-## promotion to complex ##
+## Promotion to complex ##
 
 _default_type(T::Type{Complex}) = Complex{Int}
+
+## Machine epsilon for complex ##
+
+eps(z::Complex{T}) where {T<:AbstractFloat} = hypot(eps(real(z)), eps(imag(z)))
```

Comment:
`hypot` is not defined till later in the system image load

So I think this should just be

```julia

function eps(z::Complex{T}) where {T<:AbstractFloat}
	x, y = eps(real(z)), eps(imag(z))
	if x < y
	   x, y = y, x
	end
	r = y/x
	x * sqrt(one(T) + r * r)
end

```